src/index.ts

```diff
@@ -349,10 +350,12 @@ export class CruxClient extends CruxPayPeer {
 
     public putAddressMap = async (newAddressMap: IAddressMapping): Promise<boolean> => {
         try {
+            const userAddressMap = newAddressMap;
```

Comment:
I don't see any benefit of reassigning the value to a new variable.

Reply:
Renaming was done to have similar taxonomy for what is being returned by `getAddressMap` and input expected in `putAddressMap` without affecting the already shared docs with people.